pyvptree/include/ISerializable.hpp

```diff
@@ -40,14 +74,21 @@ struct SerializedState {
         }
         return sum % 256;
     }
-    void buildChecksum() { checksum = calculate_check_sum(data); }
 };
 
 class ISerializable {
     public:
     virtual SerializedState serialize() const = 0;
     virtual void deserialize(const SerializedState &state) = 0;
 };
+
+std::ostream &operator<<(std::ostream &os, const SerializedState &state) {
+    for (int i = 0; i < state.data.size(); i++) {
```

Comment:
should this be `size_t`?

Reply:
Fixed on 81a53ff4fbc34b9916d4bb0df767ccbb01506712.